keras/backend/common.py

```diff
@@ -145,6 +145,29 @@ def set_image_data_format(data_format):
     _IMAGE_DATA_FORMAT = str(data_format)
 
 
+class BackendFunction(object):
```

Comment:
What is insufficient about `keras.layers.Lambda` for this purpose? Can you motivate this change a little more?